pkg/ddevapp/config.go

```diff
@@ -257,36 +257,14 @@ func (app *DdevApp) WriteConfig() error {
 		return err
 	}
 
-	// Allow project-specific post-config action
-	err = appcopy.PostConfigAction()
-	if err != nil {
-		return err
-	}
-
-	// Write example Dockerfiles into build directories
-	contents := []byte(`
-## #ddev-generated
-## You can copy this Dockerfile.example to Dockerfile to add configuration
-## or packages or anything else to your webimage
-## These additions will be appended last to DDEV's own Dockerfile
-## See examples here https://ddev.readthedocs.io/en/stable/users/extend/customizing-images/#adding-extra-dockerfiles-for-webimage-and-dbimage
-# RUN echo "Built on $(date)" > /build-date.txt
-`)
-
-	err = WriteImageDockerfile(app.GetConfigPath("web-build")+"/Dockerfile.example", contents)
+	// The .ddev directory may still need to be populated, especially in tests
+	err = PopulateExamplesCommandsHomeadditions(appcopy.Name)
```

Comment:
I don't know why we didn't have this function here, I find it quite valuable to populate the app assets on `ddev config` so that people can change them before `ddev start`.